dask/array/core.py

```diff
@@ -859,6 +861,8 @@ def store(sources, targets, lock=True, regions=None, compute=True, **kwargs):
 
     >>> store([x, y, z], [dset1, dset2, dset3])  # doctest: +SKIP
     """
+    from ..delayed import Delayed
```

Comment:
Should be able to move this import to the file level.

Reply:
I thought that too, but had seen that `store` was doing this import internally previously for the `compute=False` case. 😕 Not sure if there was a reason for it or if it still mattered.

Reply:
Should add there are a bunch of similar imports littered throughout the code.

Reply:
Sometimes local imports are necessary. This also isn't a huge issue, just feels like if you're touching this code it's worth cleaning it up here at this time.

Reply:
Agreed. Just wanted to make sure I wasn't missing anything before doing it. Have made this change.